test/blackbox-tests/test-cases/merlin-tests/run.t

```diff
@@ -0,0 +1,33 @@
+  $ $JBUILDER build @print-merlins -j1 --display short --root .
+      ocamldep sanitize-dot-merlin/sanitize_dot_merlin.ml.d
+        ocamlc sanitize-dot-merlin/.sanitize_dot_merlin.eobjs/sanitize_dot_merlin.{cmi,cmo,cmt}
+      ocamlopt sanitize-dot-merlin/.sanitize_dot_merlin.eobjs/sanitize_dot_merlin.{cmx,o}
+      ocamlopt sanitize-dot-merlin/sanitize_dot_merlin.exe
+  sanitize_dot_merlin alias print-merlins
```

Comment:
IIUC, two merlin files are merged in this output. It would be better to print the name before each of them.